configs/setups/awicm3/awicm3.yaml

```diff
@@ -299,15 +317,52 @@ fesom:
         choose_general.version:
                 "v3.0":
                         namelist_dir: "${esm_namelist_dir}/fesom2/2.0/awicm3/v3.0/"
+                        add_namelist_changes:
+                                namelist.oce:
+                                        oce_tra:
+                                                surf_relax_s: "${surf_relax_s}"
```

Comment:
```suggestion
```
I think this is not needed as it is part of the general `namelist_changes` and it seems you wanted for all the 4 different versions.

Reply:
The problem is that this option used to be in namelist.oce, but is now in master in namelist.tra. Thus a general option is no longer possible. We have to state per version where we want that change made.

Reply:
That's an annoying design decision, but so be it.